### ISSUE TITLE
Massive conflicts after upgrading React-Native to 0.46

### ISSUE THREAD

**Issue by xareelee** [storybooks/storybook#1454]
I just upgrade React-Native from 0.45.1 to 0.46.1. After upgrading, the stroyshots cases failed massively. All `<View/>`, `<Text/>`, `<Image/>` are replaced with `</>` components in the result of the jest.

Need I do something to fix this?

Anyone encountered this?

A sample of the jest failed results with storyshots is:

![screen shot 2017-07-13 at 09 32 33](https://user-images.githubusercontent.com/804308/28146990-1de87d58-67b0-11e7-91d3-471162ba13ab.png)

I have hundred tests failed with the same problem.

**Comment by shilman** [comment]
Yowza! @xareelee there's a PR that might help: https://github.com/storybooks/storybook/pull/1424

Maybe try that out in the short term, and if not one of us will dig into it soon.

**Comment by xareelee** [comment]
@shilman Thanks for your reply. I'm not sure how to try the commit. I use 3 libs from storybook:

```
  "devDependencies": {
    "@storybook/addon-knobs": "^3.1.8",
    "@storybook/addon-storyshots": "^3.1.8",
    "@storybook/react-native": "^3.1.8",
    ...
  }
```

Do I need to remove them in the `package.json`, and use [`yarn add --dev <git remote url>#<branch/commit/tag>`](https://yarnpkg.com/lang/en/docs/cli/add/) to install a specific commit?

I have no idea how to find the commit url in the pull request page https://github.com/storybooks/storybook/pull/1424. Can you help me out?

BTW, a sample of the jest failed results with storyshots is:

![screen shot 2017-07-13 at 09 32 33](https://user-images.githubusercontent.com/804308/28146990-1de87d58-67b0-11e7-91d3-471162ba13ab.png)

I have hundred tests failed with the same problem.

**Comment by xareelee** [comment]
@shilman Don't bother. 😄 

This issue has been resolved by upgrading React-Native from `0.46.1` to `0.46.2`.

**Comment by shilman** [comment]
Glad to hear you fixed it and thanks for posting the fix!
